models/balancer/ethereum/balancer_vebal_slopes.sql

```diff
@@ -0,0 +1,149 @@
+{{
+    config(
+        alias='vebal_slopes',
+        partition_by = ['block_date'],
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['block_time', 'wallet_address'],
+        post_hook='{{ expose_spells(\'["ethereum"]\',
+                                    "project",
+                                    "balancer",
+                                    \'["markusbkoch", "mendesfabio", "stefenon"]\') }}'
+    ) 
+}}
+
+
+WITH base_locks AS (
+        SELECT d.provider, ts AS locked_at, locktime AS unlocked_at, ts AS updated_at
+        FROM {{ source('balancer_ethereum', 'veBAL_call_create_lock') }} l
+        JOIN {{ source('balancer_ethereum', 'veBAL_evt_Deposit') }} d
+        ON d.evt_tx_hash = l.call_tx_hash
+        {% if is_incremental() %}
+        WHERE evt_block_time >= DATE_TRUNC('day', NOW() - interval '1 week')
+        {% endif %}
```

Comment:
I think I missed some filters here.
```suggestion
        SELECT d.provider, ts AS locked_at, locktime AS unlocked_at, ts AS updated_at
        FROM {{ source('balancer_ethereum', 'veBAL_call_create_lock') }} l
        JOIN {{ source('balancer_ethereum', 'veBAL_evt_Deposit') }} d
        ON d.evt_tx_hash = l.call_tx_hash
        {% if is_incremental() %}
        WHERE d.evt_block_time >= DATE_TRUNC('day', NOW() - interval '1 week')
        AND l.call_block_time >= DATE_TRUNC('day', NOW() - interval '1 week')
        {% endif %}
```